spec/models/project_mediaflux_spec.rb

```diff
@@ -50,6 +49,18 @@
         end
       end
 
+      describe "quota", connect_to_mediaflux: true do
+        it "adds a quota when it creates a project in mediaflux" do
+          project = FactoryBot.create(:project_with_doi)
+          described_class.create!(project: project, session_id: current_user.mediaflux_session)
+          metadata = Mediaflux::Http::AssetMetadataRequest.new(
+            session_token: current_user.mediaflux_session, 
+            id: project.mediaflux_id
+            ).metadata
+          expect(metadata[:quota_allocation]).not_to be_empty
```

Comment:
@JaymeeH could you check for `500 GB` here? ( I think that is the default allocation)